dev-python/frozendict/files/frozendict-2.3.4-optional-extension.patch

```diff
@@ -0,0 +1,22 @@
+frozendict does not implement native extensions for all available python
+versions, which makes the build process fail even if pure python version
+is available. This can be avoided by setting optional = True, which will
+make the installation process to continue even with failing build of
+native extension, the failing extension will simply not be installed in
+that case.
+
+diff --git a/setup.py b/setup.py
+index 0325383..ba8a800 100755
+--- a/setup.py
++++ b/setup.py
+@@ -120,6 +120,7 @@ ext_modules.append(setuptools.Extension(
+     include_dirs = cpython_include_dirs,
+     extra_compile_args = extra_compile_args,
+     undef_macros = undef_macros,
++    optional = True,
```

Comment:
`optional` really sucks, let's not do that, as it will also hide valid build failures.

Reply:
I think your patch from https://github.com/Marco-Sulla/python-frozendict/pull/69 is much better, and also short ;-).

Reply:
Ok, yes, you are right. I was searching about setuptools & extensions and have found this flag and wanted to try it. Anyway, I see how it sucks, I don't know why I pushed it at the end, sorry :)

I have force pushed update with patch from https://github.com/Marco-Sulla/python-frozendict/pull/69.